---
title: Whole genome sequencing reveals rare off‐target mutations and considerable inherent genetic or/and somaclonal variations in CRISPR/Cas9‐edited cotton plants
weight: 13
---

The CRISPR/Cas9 system has been extensively applied for crop improvement. However, our understanding of Cas9 specificity is very limited in Cas9‐edited plants. To identify on‐ and off‐target mutation in an edited crop, we described whole genome sequencing (WGS) of 14 Cas9‐edited cotton plants targeted to three genes, and three negative (Ne) control and three wild‐type (WT) plants. In total, 4188–6404 unique single‐nucleotide polymorphisms (SNPs) and 312–745 insertions/deletions (indels) were detected in 14 Cas9‐edited plants compared to WT, negative and cotton reference genome sequences. Since the majority of these variations lack a protospacer‐adjacent motif (PAM), we demonstrated that the most variations following Cas9‐edited are due either to somaclonal variation or/and pre‐existing/inherent variation from maternal plants, but not off‐target effects. Of a total of 4413 potential off‐target sites (allowing ≤5 mismatches within the 20‐bp sgRNA and 3‐bp PAM sequences), the WGS data revealed that only four are bona fide off‐target indel mutations, validated by Sanger sequencing. Moreover, inherent genetic variation of WT can generate novel off‐target sites and destroy PAMs, which suggested great care should be taken to design sgRNA for the minimizing of off‐target effect. These findings suggested that CRISPR/Cas9 system is highly specific for cotton plants.

---
> Li, J. , Manghwar, H. , Sun, L. , Wang, P. , Wang, G. , Sheng, H. , Zhang, J. , Liu, H. , Qin, L. , Rui, H. , Li, B. , Lindsey, K. , Daniell, H. , Jin, S. and Zhang, X. (2019), Whole genome sequencing reveals rare off‐target mutations and considerable inherent genetic or/and somaclonal variations in CRISPR/Cas9‐edited cotton plants. *Plant Biotechnol J*, 17: 858-868

---
原文链接：https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.13020